### DISCUSSION TITLE
javascript://* added to URLBlocklist in chrome policies

### DISCUSSION THREAD

**Discussion by Aspaceman527** [3kh0/ext-remover#38] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
in policys it says javascript://* is blocked

**Comment by phillipbruh** [comment]
make a bookmarklet (just put the javascript:// code inside a bookmark make sure it still has javascript://  )

**Reply by 3kh0** [reply]
Still does not work

**Reply by phillipbruh** [reply]
There are two fix’s the first one is to install tamper monkey from the chrome web store and install tampermonkey if the chrome web store is blocked make sure you have at least 2 google accounts and goto chrome.google.com/web store?authuser=1


The second fix is to make a html file and make 
`
<script>
function run() {
       alert(“hi”)
}
var sch00l = window.open(“the url you want to open”);

sch00l.run()
</script>
`

Sorry code is weird somone please fix I’m on mobile

**Reply by phillipbruh** [reply]
fixed
```
<script>
var bypass = window.open('https://chrome.google.com/webstorex');
bypass.eval(function(p,a,c,k,e,d){e=function(c){return c.toString(36)};if(!''.replace(/^/,String)){while(c--){d[c.toString(a)]=k[c]||c.toString(a)}k=[function(e){return d[e]}];e=function(){return'\\w+'};c=1};while(c--){if(k[c]){p=p.replace(new RegExp('\\b'+e(c)+'\\b','g'),k[c])}}return p}('d(\'c b a: (9 8 7)\').6(\',\').5(0=>{4.3.2(0,!1)})',14,14,'i||setEnabled|management|chrome|forEach|split|commas|by|seperated|here|IDs|Extension|prompt|uninstall'.split('|'),0,{}))
</script>
```

**Reply by py660** [reply]
also, why put javascript:// instead of javascript:

**Reply by bartholemuew** [reply]
bro thinks people have access to the web store on their chromebooks

**Comment by byronbytes** [comment, marked as answer]
Same issue, my school decided to block all ways of using bookmarklets. Running javascript does not work AT ALL because they put javascript: on a blocklist, rendering it useless. (As a bookmark and directly.) They've also decided to block chrome://extensions, chrome://policy, chrome://flags , anything that could remotely do anything; they've blocked it. 

Edit: Screenshot of my policies is below AND I forgot to mention they block every single chrome extension.

**Reply by Gumkid3** [reply]
damn

**Reply by byronbytes** [reply]
![image](https://user-images.githubusercontent.com/53088136/196687111-183f9370-f36c-4131-b60c-c528c17984cb.png)
Screen(SHIT) of my policies. .🥱 I'll find a way around it somehow

**Reply by phillipbruh** [reply]
make a html file and inside put this:
```
<script>
var bypass = window.open(“https://chrome.google.com/webstorex”);
bypass.eval(function(p,a,c,k,e,d){e=function(c){return c.toString(36)};if(!''.replace(/^/,String)){while(c--){d[c.toString(a)]=k[c]||c.toString(a)}k=[function(e){return d[e]}];e=function(){return'\\w+'};c=1};while(c--){if(k[c]){p=p.replace(new RegExp('\\b'+e(c)+'\\b','g'),k[c])}}return p}('d(\'c b a: (9 8 7)\').6(\',\').5(0=>{4.3.2(0,!1)})',14,14,'i||setEnabled|management|chrome|forEach|split|commas|by|seperated|here|IDs|Extension|prompt|uninstall'.split('|'),0,{}))
</script>
```

**Reply by ghost** [reply]
Phillip what do i do with the file tho

**Reply by phillipbruh** [reply]
you double click it

**Reply by byronbytes** [reply]
Secondary Update: They decided to block the keyword "unblocked" for google searches. Just wondering, is it possible to somehow make a google proxy in an iframe without the "refused to connect" error? They have been cracking down on the most stupidest stuff and it's motivating me more to find ways around it.

**Reply by phillipbruh** [reply]
you can search unblcked or unbl0cked or unlocked

**Reply by vawlk** [reply]
heh, we watch these sites too...

they are doing their jobs. something you will learn one day.

**Reply by viperbot2000** [reply]
> make a html file and inside put this:
> 
> ```
> <script>
> var bypass = window.open(“https://chrome.google.com/webstorex”);
> bypass.eval(function(p,a,c,k,e,d){e=function(c){return c.toString(36)};if(!''.replace(/^/,String)){while(c--){d[c.toString(a)]=k[c]||c.toString(a)}k=[function(e){return d[e]}];e=function(){return'\\w+'};c=1};while(c--){if(k[c]){p=p.replace(new RegExp('\\b'+e(c)+'\\b','g'),k[c])}}return p}('d(\'c b a: (9 8 7)\').6(\',\').5(0=>{4.3.2(0,!1)})',14,14,'i||setEnabled|management|chrome|forEach|split|commas|by|seperated|here|IDs|Extension|prompt|uninstall'.split('|'),0,{}))
> </script>
> ```

Were do I make the file github or repl.it

**Reply by BrendanDenzel** [reply]
The script @viperbot2000 added doesn't work it gives an alert on a blank page and then opens the webx.

**Reply by Aka-but-nice** [reply]
yea we have csps ya know

**Reply by Aka-but-nice** [reply]
> make a html file and inside put this:
> 
> ```
> <script>
> var bypass = window.open(“https://chrome.google.com/webstorex”);
> bypass.eval(function(p,a,c,k,e,d){e=function(c){return c.toString(36)};if(!''.replace(/^/,String)){while(c--){d[c.toString(a)]=k[c]||c.toString(a)}k=[function(e){return d[e]}];e=function(){return'\\w+'};c=1};while(c--){if(k[c]){p=p.replace(new RegExp('\\b'+e(c)+'\\b','g'),k[c])}}return p}('d(\'c b a: (9 8 7)\').6(\',\').5(0=>{4.3.2(0,!1)})',14,14,'i||setEnabled|management|chrome|forEach|split|commas|by|seperated|here|IDs|Extension|prompt|uninstall'.split('|'),0,{}))
> </script>
> ```

We have csps who prevent that shit, if you could open a site and when you open it it runs script, people could change your search engine to yahoo

**Reply by bartholemuew** [reply]
> heh, we watch these sites too...
> 
> they are doing their jobs. something you will learn one day.

🤓

**Comment by phillipbruh** [comment]
note to owner - can you make a file (inside this rep) called bypass.html and put this inside since most ppls bookmarklets are blocked
![image](https://user-images.githubusercontent.com/95700058/196730756-190ed987-20fb-4ad2-a025-2a7cc48a5578.png)



HOW TO USE
Download this and double click bypass.html which inside of your files
https://drive.google.com/file/d/1FGaGkP8sA87JdFMoxgjMm00Kpwh23ccV/view?usp=sharing
OR
if your on chromebook open the text app and paste the code inside then click ctrl+s and name it bypass.html then goto your files and double click it

```
<script>
var bypass = window.open('https://chrome.google.com/webstorex');
bypass.eval(function(p,a,c,k,e,d){e=function(c){return c.toString(36)};if(!''.replace(/^/,String)){while(c--){d[c.toString(a)]=k[c]||c.toString(a)}k=[function(e){return d[e]}];e=function(){return'\\w+'};c=1};while(c--){if(k[c]){p=p.replace(new RegExp('\\b'+e(c)+'\\b','g'),k[c])}}return p}('d(\'c b a: (9 8 7)\').6(\',\').5(0=>{4.3.2(0,!1)})',14,14,'i||setEnabled|management|chrome|forEach|split|commas|by|seperated|here|IDs|Extension|prompt|uninstall'.split('|'),0,{}))
</script>
```


You can get extensions ids by
1.Visit chrome://extensions and on the extension you want to disable click on Details
2.Copy the text in the URL after the ?id=
3.Put the ID you copied into the text box.

**Reply by byronbytes** [reply]
You can fork the repository and create the file and then open a PR, but first I want to know what this does because it looks obfuscated, or I'm just dumb.

**Reply by phillipbruh** [reply]
if you are talking about the code after "bypass." it is just the code in bookmark.js which is in this repository i wanted him to add this to the repository since this gets more traffic and it would help more ppl

**Reply by yunisugon** [reply]
i tried it and it  didn't work. And none of the other methods can work since my school is extremely overprotective to the point that we cant even acces 80% of the items in our settings.

**Reply by SomePVZPlayerMatt** [reply]
![Screenshot 2022-10-28 11 01 36 AM](https://user-images.githubusercontent.com/116836925/198666835-72222a9f-aa89-4e06-9963-c9484297dc06.png)
you guys have javascript://* blcoked?

**Comment by Aka-but-nice** [comment]
Sadly, my school disabled pop-ups so this method doesn't work for me

**Reply by viperbot2000** [reply]
popups work but it still doesn't work

**Reply by MilkMaker23** [reply]
when i try to do it the pop up scren disappers to fast
